src/main/java/uk/gov/companieshouse/api/accounts/model/filing/Data.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.companieshouse.api.accounts.model.filing;
+
+import java.util.Date;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Data {
+
+    @JsonProperty("links")
+    private List<Link> links;
+
+    @JsonProperty("period_end_on")
+    @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="yyyy-MM-dd")
+    private Date periodEndOn;
+
+    public List<Link> getLinks() {
```

Comment:
Can we use Lombok to remove the need for getters and setters? Add `@Getter` and `@Setter` at the class level. This will tidy up models, and has been agreed by the leads and architects.

Reply:
i will do it in different commit since I will need to change other getters and setters.

Reply:
Cool by me! But after fighting the case yesterday, you *are* going to use it ;)

Reply:
cool by me too ;)